cmd/ingester/app/flags_test.go

```diff
@@ -51,4 +54,15 @@ func TestFlagDefaults(t *testing.T) {
 	assert.Equal(t, DefaultGroupID, o.GroupID)
 	assert.Equal(t, DefaultParallelism, o.Parallelism)
 	assert.Equal(t, DefaultEncoding, o.Encoding)
+	assert.Equal(t, DefaultDeadlockInterval, o.DeadlockInterval)
+}
+
+func TestUnparsableDeadlockIntervalFlag(t *testing.T) {
```

Comment:
Shouldn't it cause an error in flags parsing instead? This is what happens when I pass `hello` as a timeout
```
$ go run ./cmd/query/main.go --cassandra.timeout=hello
Error: invalid argument "hello" for "--cassandra.timeout" flag: time: invalid duration hello
Usage:
  jaeger-query [flags]
...
```